src/domain/service/auth.ts

```diff
@@ -1,9 +1,19 @@
 import jwt from 'jsonwebtoken';
 
 /**
- * Jwt service
+ * Payload interface
```

Comment:
Describe the reason and behaviour of this interface and move it to the Entities if it will be used as important part of a logic

Reply:
Moved and improved description